ultralytics/engine/exporter.py

```diff
@@ -1011,10 +1012,11 @@ def _add_tflite_metadata(self, file):
         """Add metadata to *.tflite models per https://www.tensorflow.org/lite/models/convert/metadata."""
         import flatbuffers
 
-        if MACOS:  # TFLite Support bug https://github.com/tensorflow/tflite-support/issues/954#issuecomment-2108570845
+        if MACOS or IS_RASPBERRYPI:  # TODO: should this be 'if ARM64:'?
```

Comment:
@lakshanthad I added a note here, maybe this should be for all ARM64 devices as the current MACOS tests use `macos-14`, which is M1. Can you test this change in a separate PR by also adding `macos-latest` (uses Intel chips) runner to tests?

Reply:
Will do and feedback @glenn-jocher